Cargo.toml

```diff
@@ -25,10 +25,10 @@ exclude = [
 ]
 keywords = ["uProtocol", "SDK", "vsomeip", "SOMEIP"]
 license = "Apache-2.0"
-name = "up-client-vsomeip-rust"
+name = "up-transport-vsomeip"
 readme = "README.md"
 repository = "https://github.com/eclipse-uprotocol/up-client-vsomeip-rust"
-rust-version = "1.72"
+rust-version = "1.74"
```

Comment:
why 1.74 rather than 1.76? To keep parity with the up-rust library?

Reply:
up-transport-zenoh-rust uses 1.74

Reply:
Got it, and what was the reasoning behind removing the toolchain file?

Reply:
cargo tarpaulin was failing at 1.74 and I need to work through why. Until then, I just removed the toolchain file :upside_down_face:

Reply:
Ah got it! FWIW, I had a similar issue with tarpaulin and it resolved itself once I upgraded to 1.76. Has something to do with tarpaulin using a feature not available until that version

Reply:
Gotcha. Zenoh will upgrade to 1.75 for their 1.0.0 release, so I was hoping it'd just resolve then. But maybe not hah